test/source/tests/browser-unit-tests/unit-ContactStore.js

```diff
@@ -83,12 +83,6 @@ BROWSER_UNIT_TEST_NAME(`ContactStore is able to search by a chunk spanning acros
       throw Error(`Expected 3 contacts to match "abcdef." but got "${test.length}"`);
     }
   }
-  {
```

Comment:
So actually instead of deleting this test, we can test that `.length === 0`.
We'd better put this into a separate test (`ContactStore.search ignores top-level domains and common domains`).
With this setup 
```
  await ContactStore.update(undefined, 'abcdef.com@abcdef.com', { pubkey: testConstants.abcdefTestComPubkey });
  await ContactStore.update(undefined, 'abcdef@test.com', { pubkey: testConstants.abcdefTestComPubkey });
  await ContactStore.update(undefined, 'test@abcdef.com', { pubkey: testConstants.abcdefTestComPubkey });
```
searching for `test` should yield one result (`test@abcdef.com`)
searching for `com` should yield one result (`abcdef.com@abcdef.com`)
searching for `test.com` should yield no results